native-rating/implementation.md

```diff
@@ -2,95 +2,100 @@
 title: Building an Implementation
 ---
 
-## Building a Native Rating implementation
+# Building a Native Rating implementation
 
-### Install the CLI
+## Setup
 
-First, ensure that you have [Node](https://nodejs.org/en/) v12+ and [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) installed. Mac users will also need the development tools installed, but you will be prompted to install them
-when you run the CLI.
+Follow the [getting started](/getting-started/installing-nodejs/) walkthrough. On the [Initialize Project](/getting-started/initialize-project/) step, choose `Carrier app` when asked "what type of app are you building" and answer `Y` when asked "will this app use native rating". When this is finished, you will have a "fully working" Carrier connect app that uses Native Rating, ready to be modified however you see fit.
 
-To install [ShipEngine Connect CLI](https://connect.shipengine.com/), run the following command from your terminal:
+## Configuration
 
-```bash
-npm install --global @shipengine/connect
-```
+Before building your implementation, it is recommended to get the basic configuration out of the way. All configuration is done per carrier and is contained in the `NativeRating` property of the [carrier metadata](/shipping/#metadata-definition).
 
-Verify that the CLI was installed correctly:
+### DefaultRateCardId
 
-```bash
-connect --version
-```
+The `DefaultRateCardId` property is the only value required to use Native Rating for your Carrier app. This rate card will be used for any customer that does not have a rate card id specified in their seller provider settings. If all rate cards are managed via Connect, then this id should match the id of one of the rate cards defined in the `RateCards` property discussed below. If this id does _not_ match a rate card defined in the `RateCards` property, then a rate card with that id should be created manually through the Native Rating management API directly or else the Native Rating system will return an error when attempting to get rates. This value can be changed at any time.
```

Comment:
Can we spell out `identifier` instead of `id`, since this is prose? Use `id` if we are referring to something specific code syntax, and not the concept.